tests/robustness/main_test.go

```diff
@@ -149,7 +149,10 @@ func runScenario(ctx context.Context, t *testing.T, s scenarios.TestScenario, lg
 		return nil
 	})
 	g.Wait()
-	return append(operationReport, append(failpointClientReport, watchReport...)...)
+
+	// we query for revision 0, which asks the hash to be computed on all keys
```

Comment:
This is not how `revision 0` works

Reply:
See https://github.com/etcd-io/etcd/pull/19680#discussion_r2017014944